### PR TITLE
Add marginBottom0 to Autosuggestion

### PR DESCRIPTION
**Pull request by cchiama**
Add marginBottom0 to Autosuggestion to pass it to parent component

### CODE REVIEW

File: lib/AutoSuggest/AutoSuggest.js [hunk 1]
```diff
@@ -122,10 +125,12 @@ class AutoSuggest extends React.Component {
                 aria-live="assertive"
                 aria-relevant="additions"
               >
-                <TextField {...getInputProps(
+                <TextField
+                  {...getInputProps(
                   { ...inputProps,
                     onBlur }
                 )}
+                  marginBottom0
```
Comment (rasmuswoelk) [inline lib/AutoSuggest/AutoSuggest.js]: It looks like this should be `marginBottom0={marginBottom0}` instead.

Comment (cchiama) [reply lib/AutoSuggest/AutoSuggest.js]: Ok. I fix the issue now. Thanks @rasmuswoelk

**Comment by cchiama** [comment]
I fix the syntax for marginBottom0 inAutosuggestion. @rasmuswoelk please as soon as you can merge the fix with matteopascucci modify also, because i I need it as soon as possible. Thanks in advance

**Comment by rasmuswoelk** [comment]
@cchiama You closed the issue – was that on purpose?

I can merge it as soon as you re-open it and add the prop in the readme for the component 👍

**Pull request closed without merging**